inventory.md

```diff
@@ -13,6 +13,10 @@ _Please add all relevant resources as a table row under the relevant section or
 * [Voluntary security attestation programmes](#voluntary-security-attestation-programmes)
 * [Other](#other)
 
+## General 'meta' guidance
+
+* [CNLL guide to CRA implementation for open soure SMEs](https://code.inno3.eu/ouvert/guide-cra/-/tree/main?ref_type=heads)
```

Comment:
While the manufacturer content is great, the steward content needs some fixes. I have sent some feedback. Until then, I believe some kind of disclaimer would be warranted.